From 95ff5d19a2c7e2924ce7963de2f1396d6b2ecbf9 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 9 Aug 2023 15:12:41 +0200
Subject: [PATCH 0540/1141] dt-bindings: interrupt-controller: stm32-exti: fix
 subnode name

The subnode named 'exti-interrupt-map' is not a pattern property
and the incorrect definition triggers error by make dtbs_check.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I48f64819a03edb8902498234ca4e0af67fcc660e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/329520
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../bindings/interrupt-controller/st,stm32-exti.yaml           | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml b/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml
index 2eff63eb3059..2dca75324274 100644
--- a/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml
+++ b/Documentation/devicetree/bindings/interrupt-controller/st,stm32-exti.yaml
@@ -46,8 +46,7 @@ properties:
 
   wakeup-parent: true
 
-patternProperties:
-  '^exti-interrupt-map$':
+  exti-interrupt-map:
     type: object
     properties:
       interrupt-map: true
-- 
2.39.2

